AxonDeepSeg/mapping_results.py

```diff
@@ -1,11 +1,7 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-from pathlib import Path
 import json
-from tqdm import tqdm
-
-import AxonDeepSeg.ads_utils
```

Comment:
@vs74 AxonDeepSeg.ads_utils is imported in every module file for bug tracking purposes, so please don't remove them even though you don't see them explicitely used in the files.

Reply:
Should I revert it ?

Reply:
No I just did so